sqlpage/templates/shell.handlebars

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="{{language}}" style="font-size: {{default font_size 18}}px">
+<html lang="{{language}}" style="font-size: {{default font_size 18}}px" {{#if html_class}}class="{{html_class}}"{{/if}}>
```

Comment:
We call it simply `class` everywhere else. Can you rename it ?

Reply:
Yes! I was considering a `body_class` too (which led to the distinction), but I later took that away. Probably not needed.

The point of having a class on the `<html>` at all I think is to assist with logical class names (such as the Shoelace custom theming I pointed to earlier) that CSS can latch on to.

Just `class` should be fine; `<html>` is the ultimate shell 👌